docs/source/en/add_new_model.md

```diff
@@ -622,339 +425,202 @@ tensor([[-0.0818,  0.2207, -0.0749, -0.0030,  0.0045, -0.1569, -0.1598,  0.0212,
+
+[Cookiecutter](https://cookiecutter.readthedocs.io/en/stable/) should have added a test file for your model. Run the test file below to make sure all common tests pass.
 
 ```bash
 pytest tests/models/brand_new_bert/test_modeling_brand_new_bert.py
 ```
 
-Having fixed all common tests, it is now crucial to ensure that all the nice work you have done is well tested, so that
-
-- a) The community can easily understand your work by looking at specific tests of *brand_new_bert*
-- b) Future changes to your model will not break any important feature of the model.
+The integration tests should be added first because they serve the same purpose as the debugging scripts you used earlier to implement the new model in Transformers. A template of those model tests, `BrandNewBertModelIntegrationTests`, was added by Cookiecutter and should be filled out. To ensure it passes, run the following command.
 
-At first, integration tests should be added. Those integration tests essentially do the same as the debugging scripts
-you used earlier to implement the model to 🤗 Transformers. A template of those model tests has already added by the
-Cookiecutter, called `BrandNewBertModelIntegrationTests` and only has to be filled out by you. To ensure that those
-tests are passing, run
+<hfoptions id="integration-test">
+<hfoption id="macOS">
 
 ```bash
 RUN_SLOW=1 pytest -sv tests/models/brand_new_bert/test_modeling_brand_new_bert.py::BrandNewBertModelIntegrationTests
 ```
 
-<Tip>
+</hfoption>
+<hfoption id="Windows">
 
-In case you are using Windows, you should replace `RUN_SLOW=1` with `SET RUN_SLOW=1`
-
-</Tip>
-
-Second, all features that are special to *brand_new_bert* should be tested additionally in a separate test under
-`BrandNewBertModelTester`/`BrandNewBertModelTest`. This part is often forgotten but is extremely useful in two
-ways:
+```bash
+SET RUN_SLOW=1 pytest -sv tests/models/brand_new_bert/test_modeling_brand_new_bert.py::BrandNewBertModelIntegrationTests
+```
 
-- It helps to transfer the knowledge you have acquired during the model addition to the community by showing how the
-  special features of *brand_new_bert* should work.
-- Future contributors can quickly test changes to the model by running those special tests.
+</hfoption>
+</hfoptions>
 
+All features unique to BrandNewBert should be tested in a separate test under `BrandNewBertModelTester/BrandNewBertModelTest`. This test is often overlooked, but it is extremely important because:
 
-**9. Implement the tokenizer**
+- it helps transfer knowledge you acquired during the process to the community by showing how the models novel features work
+- future contributors can quickly test changes to the model by running these special tests
 
-Next, we should add the tokenizer of *brand_new_bert*. Usually, the tokenizer is equivalent to or very similar to an
-already existing tokenizer of 🤗 Transformers.
+## Implement tokenizer
```

Comment:
IMO we should recommend adding fast tokenizer / asking me or @itazap how to convert to `PreTrainedTokenizerFast`!